opentasks/src/main/java/org/dmfs/tasks/widget/LocationFieldView.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2013 Marten Gajda <marten@dmfs.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * 
+ */
+
+package org.dmfs.tasks.widget;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.GestureDetector;
+import android.view.MotionEvent;
+import org.dmfs.tasks.utils.MapOpener;
+
+
+/**
+ * A widget that shows the location. When clicked, it opens maps.
+ *
+ * @author Gabor Keszthelyi
+ */
+public class LocationFieldView extends TextFieldView
+{
+    private GestureDetector mGestureDetector;
+    private MapOpener mMapOpener;
+
+
+    public LocationFieldView(Context context)
+    {
+        super(context);
+        init();
+    }
+
+
+    public LocationFieldView(Context context, AttributeSet attrs)
+    {
+        super(context, attrs);
+        init();
+    }
+
+
+    public LocationFieldView(Context context, AttributeSet attrs, int defStyle)
+    {
+        super(context, attrs, defStyle);
+        init();
+    }
+
+
+    private void init()
+    {
+        mMapOpener = new MapOpener(getContext());
+        mGestureDetector = new GestureDetector(getContext(), new GestureDetector.SimpleOnGestureListener()
```

Comment:
I spent about 2 hours trying to make the clickable thing work and I couldn't. There is of course a chance that I missed something, so let me know if I should check again and spend like 1 hour more with it.
(I tried with `clickable`, `focusable` combinations on parent and child views and some other suggestion I found on StackOverflow.)